### DISCUSSION TITLE
Display Error Pages

### DISCUSSION THREAD

**Discussion by GodsITguy** [tinyproxy/tinyproxy#531] [Q&A]
I'm just learning how to setup and got it working, but I noticed an error page section of the .conf file that I can't get working. When I got to a blocked page, it just says 'the proxy server is refusing connections", I created basic error pages, 400, 502, 503,403 and put them in the path specified. Any tips. My goals is to show a page that tells the user the page is blocked by proxy and contact support if they'd like it unblocked with justification.

**Comment by rofl0r** [comment]
other error page (code)s work for you ? how do you block ?
